build.gradle

```diff
@@ -42,6 +42,7 @@ allprojects {
         tasks.withType(JavaCompile) {
             options.compilerArgs += ['-Werror', '-Xlint:deprecation']
             options.errorprone {
+                enabled = false
```

Comment:
this can be reenabled once we release this pr and pick it up - otherwise we fail because of the break